ols/src/prompts/prompts.py

```diff
@@ -50,87 +50,87 @@
 Response:
 """
 
-SUMMARIZATION_TEMPLATE = """
-The following context contains several pieces of documentation. Please answer the user's question based on this context.
-Documentation context:
-{context_str}
-
-User query:
-{query_str}
-
-Summary:
-
-"""
-
-SUMMARY_TASK_BREAKDOWN_TEMPLATE = (
-    """
-The following documentation contains a task list. Your job is to extract the list of tasks. """
-    """If the user-supplied query seems unrelated to the list of tasks, please reply that you do not know what to do with the query and the summary documentation. """
-    """Use only the supplied content and extract the task list.
-
-Summary document:
-{context_str}
-
-User query:
-{query_str}
-
-What are the tasks?
-"""
-)
-
-TASK_PERFORMER_PROMPT_TEMPLATE = """
-Instructions:
-- You are a helpful assistant.
-- You are an expert in Kubernetes and OpenShift.
-- Respond to questions about topics other than Kubernetes and OpenShift with: "I can only answer questions about Kubernetes and OpenShift"
-- Refuse to participate in anything that could harm a human.
-- Your job is to look at the following description and provide a response.
-- Base your answer on the provided task and query and not on prior knowledge.
-
-TASK:
-{task}
-QUERY:
-{query}
-
-Question:
-Does the above query contain enough background information to complete the task? Provide a yes or no answer with explanation.
-
-Response:
-"""
-
-TASK_REPHRASER_PROMPT_TEMPLATE = """
-Instructions:
-- You are a helpful assistant.
-- Your job is to combine the information from the task and query into a single, new task.
-- Base your answer on the provided task and query and not on prior knowledge.
-
-TASK:
-{task}
-QUERY:
-{query}
-
-Please combine the information from the task and query into a single, new task.
-
-Response:
-"""
-
-YES_OR_NO_CLASSIFIER_PROMPT_TEMPLATE = """
-Instructions:
-- determine if a statement is a yes or a no
-- return a 1 if the statement is a yes statement
-- return a 0 if the statement is a no statement
-- return a 9 if you cannot determine if the statement is a yes or no
-
-Examples:
-Statement: Yes, that sounds good.
-Response: 1
-
-Statement: No, I don't think that is wise.
-Response: 0
-
-Statement: Apples are red.
-Response: 9
-
-Statement: {statement}
-Response:
-"""
+# SUMMARIZATION_TEMPLATE = """
```

Comment:
you can just delete all of these.  If we find ourselves implementing functionality where we want them again we're probably going to want to start fresh w/ the prompt text anyway, there was no special research effort invested in coming up with these particular prompt templates.